Packs/CVE_2025_31324/Playbooks/playbook-CVE-2025-31324_-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -0,0 +1,2835 @@
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "28"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: Potential_suspicious_files
+      value:
+        simple: ${XQLPossibleMaliciousFiles.results.action_file_path}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2580.75,
+          "y": 2739
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "95":
+    id: "95"
+    taskid: 75337bfd-90ec-4ba5-8da8-635868b79611
+    type: regular
+    task:
+      id: 75337bfd-90ec-4ba5-8da8-635868b79611
+      version: -1
+      name: Manual Action - Quarantine
+      description: "As a part of the playbook, we have determined potential webshells that might be in the same directory *\"...servlet_jsp\\irj\\....\"*.\nSince we can't determine with strong confidence that those files are webshells, we suggest that you investigate them and quarantine as needed. \n\nList of files:\n${Potential_suspicious_files}\n\nList of agent IDs:\n${alert.agentid}\n${alert.agentsid}"
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "31"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4626,
+          "y": 3214
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "96":
+    id: "96"
+    taskid: b2145507-a138-4ce4-b75f-153e80be66b0
+    type: condition
+    task:
+      id: b2145507-a138-4ce4-b75f-153e80be66b0
+      version: -1
+      name: Is there a file path?
+      description: Is there a alert.filepath in context data.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "28"
+      "yes":
+      - "106"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: PaloAltoNetworksXDR.OriginalAlert.messageData.dynamicAnalysis.internals.attributes.file_path
+            iscontext: true
+          right:
+            value: {}
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: incident.xdralerts.agentostype
```

Comment:
same here, didn't see that you mapped those on your FoundIncident key... ( or set it else were)
I assume it won't work